compiler/parser/src/error.rs

```diff
@@ -54,8 +54,8 @@ impl fmt::Display for LexicalErrorType {
             LexicalErrorType::DuplicateArgumentError(arg_name) => {
                 write!(f, "duplicate argument '{arg_name}' in function definition")
             }
-            LexicalErrorType::DuplicateKeywordArgumentError => {
-                write!(f, "keyword argument repeated")
+            LexicalErrorType::DuplicateKeywordArgumentError(arg_name) => {
+                write!(f, "keyword argument '{arg_name}' repeated")
```

Comment:
Here is the correct form, according to `test_syntax.py`:

```suggestion
                write!(f, "keyword argument repeated: {arg_name}")
```